includes/core.php

```diff
@@ -1267,9 +1267,9 @@ function get_available_social_icons() {
 	 *
 	 * @since 0.1.0
 	 *
-	 * @param array $social_icons Array containings the supported social icons.
+	 * @param array $social_icons Array containing the supported social icons.
 	 */
-	return (array) apply_filters( 'go_avaliable_social_icons', $social_icons );
+	return (array) apply_filters( 'go_available_social_icons', $social_icons );
```

Comment:
This needs attention.

Reply:
I am happy to send this PR.

Please see the other two https://github.com/godaddy-wordpress/go/pulls/szepeviktor

Reply:
@EvanHerman Friendly ping 🏓